packages/renderer/MdToHtml/rules/tableHorizontallyScrollable.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+	plugin: (markdownIt: any) => {
+		markdownIt.renderer.rules.table_open = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
+			if (tokens[idx].map) {
+				const line = tokens[idx].map[0];
+				const lineEnd = tokens[idx].map[1];
+				tokens[idx].attrJoin('class', 'maps-to-line');
+				tokens[idx].attrSet('source-line', `${line}`);
+				tokens[idx].attrSet('source-line-end', `${lineEnd}`);
+			}
```

Comment:
Why is this necessary?

Reply:
copy from packages/renderer/MdToHtml/rules/source_map.ts,  I will remove it.